### PR TITLE
Add submittedBy

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding submitted by

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit test

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/filingpackage/mapper/FilingPackageMapper.java [hunk 1]
```diff
@@ -26,6 +27,7 @@
     @Mapping(target = "court.agencyId", source = "court.locationId")
     @Mapping(target = "court.locationDescription", source = "court.locationDescription")
     @Mapping(target = "court.classDescription", source = "court.classDescription")
+    @Mapping(target = "submittedBy",  expression = "java(java.text.MessageFormat.format(\"{0} {1}\", filingPackage.getFirstName(), filingPackage.getLastName()))")
```
Comment (alexjoybc) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/filingpackage/mapper/FilingPackageMapper.java:30]: maybe add an account object

**Review by alexjoybc — CHANGES_REQUESTED** [review]
change the submittedBy for an account

File: src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/EfilingStatusServiceDemoImpl.java [hunk 2]
```diff
@@ -35,6 +35,7 @@ private ReviewFilingPackage createReviewPackage() {
         ReviewFilingPackage reviewFilingPackage = new ReviewFilingPackage();
         reviewFilingPackage.setFirstName("Han");
         reviewFilingPackage.setLastName("Solo");
+        reviewFilingPackage.setFilingCommentsTxt("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
```
Comment (KDNeufeld) [inline src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/EfilingStatusServiceDemoImpl.java:38]: This is good. Thanks.  
I noticed that the court.fileNumber is null in the sample api data.  Can you also provide a fileNumber?

ie, line 52:
reviewCourt.setFileNumber(42);

Comment (TayGov) [reply src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/EfilingStatusServiceDemoImpl.java:38]: 👍

**Review by alexjoybc — APPROVED** [review]
Looks good thanks @TayGov

**Pull request merged**